classes/acrn-sblimage.bbclass

```diff
@@ -0,0 +1,88 @@
+inherit python3native
+DEPENDS += "slimboot-tools-native"
+
+MB_DEPENDENCY    ?= ""
+MB_MC_DEPENDENCY ?= ""
+MB_ACRN_BINARY   ?= "${TMPDIR}/deploy/images/${MACHINE}/acrn.32.out"
+MB_ACRN_CMDLINE  ?= ""
+MB_ACRN_MODULES  ?= "${TOPDIR}/conf/linux.txt;${TMPDIR}/deploy/images/${MACHINE}/${KERNEL_IMAGETYPE}"
```

Comment:
We should probably use `DEPLOY_DIR_IMAGE` instead of `${TMPDIR}/deploy/images/${MACHINE}` in case someone has overriden that. Same for MB_ACRN_BINARY

Reply:
got it. thanks